Runtime/BlockadeLabsSkyboxMesh.cs

```diff
@@ -22,6 +20,15 @@ public enum MeshDensity
     [ExecuteAlways, RequireComponent(typeof(MeshRenderer)), RequireComponent(typeof(MeshFilter))]
     public class BlockadeLabsSkyboxMesh : MonoBehaviour
     {
+        [SerializeField]
+        private SkyboxAI _skyboxMetadata;
+
+        public SkyboxAI SkyboxAsset
+        {
+            get => _skyboxMetadata;
+            internal set => _skyboxMetadata = value;
```

Comment:
should probably be public

Reply:
I think it'll be ok to assign in the inspector bc we can assign different things, but let's keep this internal for now.